docs/install-upgrade/build-wiring.md

```diff
@@ -279,3 +282,322 @@ The dotted line in the diagram shows the traffic flow for remote peering. The tr
 
 A VPC loopback is a physical cable with both ends plugged into the same switch, suggested but not required to be the adjacent ports. This loopback allows two different VPCs to communicate with each other. This is due to a Broadcom limitation.
 
+## Sample Wiring Diagram
+
+The YAML listing below shows a complete wiring diagram. It illustrates how switches
+from a single vendor can be arranged to form a fabric. There are no IP
+addresses or ASN numbers in this listing, the controller creates those as part
+of creating the fabric. To physically connect this topology, 16 cables are
+needed for the fabric links, 8 cables are needed for the loop back connections.
+Additional cables are needed to connect servers into the fabric. 
+
+``` {.yaml .annotate linenums="1" title="wiring_diagram.yaml"}
+
+#
```

Comment:
Nit (not blocking): No need for that blank line

```suggestion
``` {.yaml .annotate linenums="1" title="wiring_diagram.yaml"}
#
```